static/js/signal.js

```diff
@@ -0,0 +1,32 @@
+window.onload = function() {
+  updateSignalLoop()
+};
+
+async function updateSignalLoop() {
+  const refreshTime =2 * 1000 
+
+  var xmlHttp = new XMLHttpRequest();
+  xmlHttp.open("GET", "/api/signal-strength", true);
+  xmlHttp.setRequestHeader("Authorization", "Basic "+btoa("admin:feathers"))
+  xmlHttp.onload = async function() {
+    if (xmlHttp.status == 200) {
+      var signalElement = document.getElementById("signal-strength");
+      signalElement.innerText = xmlHttp.response;
+      await sleep(refreshTime);
+    } else {
+      console.log("status:", xmlHttp.status);
+      console.log("response:", xmlHttp.response);
+      await sleep(refreshTime);
+    }
+    updateSignalLoop(); 
+  }
+  xmlHttp.onerror = function(err) {
+    console.log('error:', err);
```

Comment:
if the request fails once, this shouldn't immediately stop updates to the UI (updateSignalLoop is never being called again).

Also, after a number of failures the signal strength UI should probably be cleared to avoid giving a false impression

Reply:
Changed to keep checking and clear signal after const clearSignalAttempts